### PR TITLE
Spsh 1172

### PR DESCRIPTION
**Pull request by AlexanderUngefug**

### CODE REVIEW

File: src/modules/person/domain/person.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export type PersonCreationParams = {
     personalnummer?: string;
     lockInfo?: LockInfo;
     isLocked?: boolean;
+    id?: number;
 };
```
Comment (marode-cap) [inline src/modules/person/domain/person.ts:38]: Could you move `id` to the top of the params, just for better readability?

File: src/console/dbseed/domain/db-seed.service.ts [hunk 2]
```diff
@@ -163,7 +165,11 @@ export class DbSeedService {
                 throw rolle;
             }
 
-            const persistedRolle: Rolle<true> = await this.rolleRepo.save(rolle);
+            if (file.overrideId) {
+                rolle.id = this.getValidUuidOrUndefined(file.overrideId);
+            }
+
```
Comment (marode-cap) [inline src/console/dbseed/domain/db-seed.service.ts:171]: In line 111 you don't use an if, could we make this the same everywhere?

**Review by marode-cap — CHANGES_REQUESTED** [review]

File: seeding/dev/01/06_personenkontext.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "entityName": "Personenkontext",
     "entities": [
-        {
+        {            
             "personId": 1,
```
[the remark this answers is not in the document]
Comment (AlexanderUngefug) [reply seeding/dev/01/06_personenkontext.json:5]: vs code automatismus bei neuer Zeile, ist entfernt

File: src/console/dbseed/domain/db-seed.service.ts (hunk 2 above)
Comment (AlexanderUngefug) [reply src/console/dbseed/domain/db-seed.service.ts:171]: yes

**Review by marode-cap — APPROVED** [review]

**Pull request merged**